rpm/ssu.spec

```diff
@@ -20,7 +20,7 @@ Requires(pre): sailfish-setup
 Requires(pre): shadow-utils
 Requires(pre): /usr/bin/groupadd-user
 Requires(postun): shadow-utils
-Requires: ssu-network-proxy
+Obsoletes: network-proxy-plugin
```

Comment:
warning: line 23: It's not recommended to have unversioned Obsoletes: Obsoletes: network-proxy-plugin

=> Obsoletes: ssu-network-proxy <= 1.0.22

Reply:
I hoped we would drop the warning at some point instead :) fixed.

Reply:
True that but this looks good.